pytest/lib/cluster.py

```diff
@@ -777,9 +777,11 @@ def spin_up_node_and_push(i, boot_node: BootNode):
     return nodes
 
 
+ROOT_DIR = str(pathlib.Path(__file__).resolve().parents[2])
+
 DEFAULT_CONFIG = {
     'local': True,
-    'near_root': os.environ.get("NEAR_ROOT", '../target/debug/'),
+    'near_root': os.environ.get("NEAR_ROOT", os.path.join(ROOT_DIR, 'target/debug')),
```

Comment:
Better still, drop `str` from initialisation of `ROOT_DIR` and use `str(ROOT_DIR / 'target/debug')` here.